MOxUnit/moxunit_util_elem2str.m

```diff
@@ -18,20 +18,54 @@
 %     to capture the output from disp in a variable; hence only the class
 %     of the first input is shown
 %
-% NNO Jan 2014
+% NNO, SCL 2014-2015
 
     if nargin<2
         max_chars=100;
     end
 
-    if moxunit_util_platform_is_octave()
-        elem_str=sprintf('<%s>\n',class(elem));
-    else
-        % Octave does not support evalc
+    % Strings are trivally supported by leaving them as-is
+    if ischar(elem)
+        % Encapsulate in quotation marks
+        elem_str=['''' elem ''''];
+
+    elseif isnumeric(elem)
+        % If the element is numeric, we can use mat2str
+        elem_str=mat2str(elem);
```

Comment:
I see the point of prettier printing of different datastructures. I really do. Indeed, I've written something like this for another toolbox: https://github.com/CoSMoMVPA/CoSMoMVPA/blob/master/mvpa/cosmo_disp.m

Also I don't have principal objections against some prettier printing of some objects - though the question is where to draw the line, how many special cases on wants to include, etc. 

But can we please discuss this pretty printing functionality separately from this PR (which is about XML output)? Apart from not printing '<' and '>' characters, the two features seem orthogonal.